mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactLoggedModelView.tsx

```diff
@@ -151,6 +155,32 @@ class ShowArtifactLoggedModelView extends Component<Props, State> {
     );
   }
 
+  validateModelForServingText(modelPath: any, servingInput?: string) {
+    if (servingInput === null) {
+      return (
+        `import mlflow\n` +
+        `from mlflow.models import validate_serving_input\n\n` +
+        `model_uri = '${modelPath}'\n\n` +
+        `# The model logged does not contain input_example, you need to manually generate a serving payload\n` +
+        `from mlflow.models import convert_input_example_to_serving_input\n` +
+        `# Replace INPUT_EXAMPLE with your own input example to the model\n` +
+        `serving_payload = convert_input_example_to_serving_input(INPUT_EXAMPLE)\n\n` +
+        `# Validate the serving payload works on the model\n` +
+        `validate_serving_input(model_uri, serving_payload)`
```

Comment:
What about using `mlflow.models.predict()` so users can avoid the dependency issues as well?

Reply:
This is a good point! The only difference is that `serving_payload` here is a json string. I think inside validate_serving_input we can add a param like `validate_pip_requirements` and call `mlflow.models.predict()` if it's true. Because I suppose not everyone installs virtualenv and if they use local env then there's no difference.

Another concern for changing this in this PR is that I think we need to add more tests to make sure this works well, since we infer the pip requirements for the user, and if it breaks user will complain 🤣 
I can file a JIRA to track this, WDYT?

Reply:
> Another concern for changing this in this PR is that I think we need to add more tests to make sure this works well

That's fair, especially given we know that the requirement inference does not always work nicely (Q3 work item)😅 Follow-up sounds good to me!